### PR TITLE
[Source-postgres] : Source operations suport for meta column 

### PR DESCRIPTION
**Pull request by akashkulk**
One bad value should not fail the sync. 
Instead, null out the offending column and quarantine the value into a `airbyte_changes.meta` column

### CODE REVIEW

File: airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/cdc/PostgresCdcConnectorMetadataInjector.java [hunk 1]
```diff
@@ -14,14 +14,28 @@
 
 public class PostgresCdcConnectorMetadataInjector implements CdcMetadataInjector<Long> {
 
+  private final String transactionTimestamp;
```
Comment (rodireich) [inline airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/cdc/PostgresCdcConnectorMetadataInjector.java:17]: is this related to meta support?

Comment (akashkulk) [reply airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/cdc/PostgresCdcConnectorMetadataInjector.java:17]: No, it's a refactor to get rid of an extra class

File: airbyte-cdk/java/airbyte-cdk/core/src/main/java/io/airbyte/cdk/db/jdbc/AbstractJdbcCompatibleSourceOperations.java [hunk 2]
```diff
@@ -28,19 +32,48 @@
...
 public abstract class AbstractJdbcCompatibleSourceOperations<Datatype> implements JdbcCompatibleSourceOperations<Datatype> {
 
+  private static final org.slf4j.Logger LOGGER = LoggerFactory.getLogger(AbstractJdbcCompatibleSourceOperations.class);
+
   /**
    * A Date representing the earliest date in CE. Any date before this is in BCE.
    */
   private static final Date ONE_CE = Date.valueOf("0001-01-01");
 
+  public AirbyteRecordData convertDatabaseRowToAirbyteRecordData(final ResultSet queryContext) throws SQLException {
```
Comment (rodireich) [inline airbyte-cdk/java/airbyte-cdk/core/src/main/java/io/airbyte/cdk/db/jdbc/AbstractJdbcCompatibleSourceOperations.java:53]: What throws an `SQLException` here?

**Review by rodireich — APPROVED** [review]
